src/main/java/org/broadinstitute/hellbender/engine/LocalReadShard.java

```diff
@@ -150,6 +172,10 @@ public boolean containsStartPosition(final Locatable loc) {
             readsIterator = new ReadsDownsamplingIterator(readsIterator, downsampler);
         }
 
+        if (postReadFilterTransformer != null) {
+            readsIterator = new ReadTransformingIterator(readsIterator, postReadFilterTransformer);
```

Comment:
In `LocusWalker` the post-readFilter transformer is applied after filtering but before downsampling, rather than after downsampling. This is admittedly less useful, but should we try to be consistent?

Reply:
Sure.